action-api/src/test/java/com/powsybl/action/EqualsActionTest.java

```diff
@@ -0,0 +1,472 @@
+                .build();
+        Action action3 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
+                .build();
+        Action action4 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withVoltageSetpoint(5.)
+                .build();
+        StaticVarCompensatorAction action5 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withVoltageSetpoint(5.)
+                .withRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void phaseTapChangerTapPositionAction() {
+        PhaseTapChangerTapPositionAction action1 = new PhaseTapChangerTapPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(true)
+                .build();
+        PhaseTapChangerTapPositionAction action2 = new PhaseTapChangerTapPositionActionBuilder().withId("id")
+                .withTransformerId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(true)
+                .build();
+        PhaseTapChangerTapPositionAction action3 = new PhaseTapChangerTapPositionActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(true)
+                .withSide(ThreeSides.ONE)
+                .build();
+        PhaseTapChangerTapPositionAction action4 = new PhaseTapChangerTapPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId2")
+                .withTapPosition(2)
+                .withRelativeValue(true)
+                .withRelativeValue(false)
+                .build();
+        PhaseTapChangerTapPositionAction action5 = new PhaseTapChangerTapPositionActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withRelativeValue(true)
+                .withTapPosition(3)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void phaseTapChangerRegulationAction() {
+        Action action1 = new PhaseTapChangerRegulationActionBuilder().withId("id")
```

Comment:
To fully cover the `equals()` method, could you add a  `.withRegulating(true)` for `action1` and `action2`.